code/modules/admin/verbs/ai_triumvirate.dm

```diff
@@ -0,0 +1,49 @@
+
+///global reference to the current theme, if there is one.
+GLOBAL_DATUM(triple_ai_controller, /datum/triple_ai_controller)
+
+/**
+ * The triple ai controller handles the admin triple AI mode, if enabled.
+ * It is first created when "Toggle AI Triumvirate" triggers it, and it can be referenced from GLOB.triple_ai_controller
+ * After it handles roundstart business, it cleans itself up.
+ */
+/datum/triple_ai_controller
+
+/datum/triple_ai_controller/New()
+	. = ..()
+	RegisterSignal(SSjob, COMSIG_OCCUPATIONS_DIVIDED, PROC_REF(on_occupations_divided))
+
+/datum/triple_ai_controller/proc/on_occupations_divided(datum/source, pure, allow_all)
+	SIGNAL_HANDLER
+
+	for(var/datum/job/ai/ai_datum in SSjob.joinable_occupations)
+		ai_datum.spawn_positions = 3
+	if(!pure)
+		for(var/obj/effect/landmark/start/ai/secondary/secondary_ai_spawn in GLOB.start_landmarks_list)
+			secondary_ai_spawn.latejoin_active = TRUE
+		qdel(src)
```

Comment:
This doesn't seem to take into account human AI trait

Reply:
it didn't before either.